samples/dotnet/kernel-syntax-examples/Skills/JiraSkill/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2826 @@
+{
+    "swagger": "2.0",
```

Comment:
How is this file different from openapi.json? Is this being referenced by the example?

Reply:
its the same file. ill remove it